apps/web/src/hooks/useCeramic.ts

```diff
@@ -1,15 +1,43 @@
-import { ComposeClient } from "@composedb/client";
 import { EthereumWebAuth, getAccountId } from "@didtools/pkh-ethereum";
 import { InjectedConnector } from "@wagmi/core";
 import { DIDSession } from "did-session";
+import { useQuery } from "react-query";
 import { useAccount } from "wagmi";
 import { trackEvent } from "../lib/analytics";
+import { composeClient } from "../lib/compose";
 
-export function useCeramic(composeClient: ComposeClient) {
+export const LOCAL_STORAGE_KEYS = {
+  DID: "denoted.ceramic.did",
+  SIGNED_RESOURCES: "denoted.ceramic.signed-resources",
+};
+
+export function useCeramic() {
   const { address } = useAccount();
 
   const connector = new InjectedConnector();
 
+  function getIsResourcesSigned(resources: string[]) {
+    if (typeof window === "undefined") {
```

Comment:
This could be executed in non-browser?

Reply:
Not when we're using local storage, or maybe I'm missing something?